README.md

```diff
@@ -37,6 +47,22 @@ public class Status {
         return this.value;
     }
 
+    /**
+     * returns true if given enum is part of the declared values.use it in your application to detect when values coming from outside of the app are not yet part of the declared values (i.e.: there is a new version of the enum that your application is not yet aware of.
+     *
+     */
+    public static Boolean isDeclaredValue(Status val) {
+        return Status.declaredValues.contains(val);
+    }
+
+    /**
+     * returns true if given enum is NOT part of the declared values.use it in your application to detect when values coming from outside of the app are not yet part of the declared values (i.e.: there is a new version of the enum that your application is not yet aware of.
+     *
+     */
+    public static Boolean isNotDeclaredValue(Status val) {
```

Comment:
Such redundant negative accessors are not idiomatic, since there is no universality. For example, java collections do not have a "isNotEmpty" accessor.

Reply:
Will keep only negative form